cmd/nuke.go

```diff
@@ -115,9 +115,7 @@ func (n *Nuke) Run() error {
 		}
 		if n.Parameters.MaxWaitRetries != 0 && n.items.Count(ItemStateWaiting, ItemStatePending) > 0 && n.items.Count(ItemStateNew) == 0 {
 			if waitingCount >= n.Parameters.MaxWaitRetries {
-				fmt.Printf("Max wait retries of %d exceeded.\n\n", n.Parameters.MaxWaitRetries)
-
-				break
+				return fmt.Errorf("Max wait retries of %d exceeded.\n\n", n.Parameters.MaxWaitRetries)
```

Comment:
I am not 100% about this. This would mean the first time this condition is reached the loop is terminated.

Reply:
I think the handling should happen in line 134, right?

Reply:
> This would mean the first time this condition is reached the loop is terminated.

This is the idea. If there are more than `MaxWaitRetries` iterations that contain resources in pending state, it prematurely exits.

> I think the handling should happen in line 134, right?

I would say both ways would be okay, but it should be consistent with Line 109.